phpunit.xml

```diff
@@ -1,5 +1,10 @@
 <?xml version="1.0"?>
-<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" colors="true" xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/11.0/phpunit.xsd">
+<phpunit
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  colors="true"
+  xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/11.0/phpunit.xsd"
+  displayDetailsOnTestsThatTriggerWarnings="true"
```

Comment:
Even if this is not a big modification, I am not fond of doing this in this issue. Could you remove these modifications from your commit, or explain why you need them?

Reply:
When running tests with warnings or deprecations there are no details to explain where these warnings are being triggered. These flags allow the cause of the warnings and deprecations to be displayed in more detail. It helps facilitate the debugging of tests more easily.